packages/page-header/README.md

```diff
@@ -7,7 +7,7 @@
 ## Installation
 
 ```bash
-npm install @availity/page-header formik yup --save
+npm install-peerdeps @availity/page-header --save
```

Comment:
s/b `npx` instead of `npm`

Reply:
https://github.com/Availity/availity-react/pull/234/commits/98f5ce276c634f854490bcbe8c9edd03cdea12bd